### PR TITLE
Feat improve ratings

### PR DESCRIPTION
**Pull request by mikeyrayvon**
adds individual committee members ratings to submissions list table.  adds list of committee members ratings to submission review template for admin users.  adds newlines to textarea data on submission review template

### CODE REVIEW

File: app/client/routes/routerAdmin.js [hunk 1]
```diff
@@ -71,16 +72,37 @@ Router.map(function() {
...
-      return {
-        application: Applications.findOne({userId: this.params.userId,}),
-      };
+      var userId = Meteor.userId();
+
+      if (Roles.userIsInRole(userId, 'admin')) {
+        return {
+          application: Applications.findOne({userId: this.params.userId,}),
+          committee: Roles.getUsersInRole('committee'),
+          ratings: Ratings.find().fetch(),
```
Comment (mixcas) [inline app/client/routes/routerAdmin.js]: why `fetch()`?
where is `ratings` used? i cant find it in the template.

File: app/client/templates/admin/submissionReview.js [hunk 2]
```diff
@@ -1,3 +1,25 @@
+Template.submissionReview.helpers({
+
+  breaklines: function (text) {
```
Comment (mixcas) [inline app/client/templates/admin/submissionReview.js]: this function names don't tell what they do. something like `formatWithBreaklines` is more descriptive. I suck at naming things tho, i think u can come with a better name.

same for `userRating`, this one just need a verb like `getUserRating`.

File: app/client/templates/admin/submissions.html [hunk 3]
```diff
@@ -35,18 +40,30 @@
               {{projectTitle}}
             {{/if}}
           </td>
-          <td>
+          <td>          
             {{status}}
           </td>
-          {{#if isAdmin}}
           <td>
             {{#if isSubmitted}}
```
Comment (mixcas) [inline app/client/templates/admin/submissions.html]: This is not working for me, the helper in `helpers/spacebars.js` is failing cuz `this.status` doesn't exist it should be `this.application.status` cuz of the way `data` is declared in the adminRouter

Comment (mixcas) [inline app/client/templates/admin/submissions.html]: This is not working for me, the helper in helpers/spacebars.js is failing cuz this.status doesn't exist it should be this.application.status cuz of the way data is declared in the adminRouter.

File: app/client/templates/admin/submissions.js [hunk 4]
```diff
@@ -1,7 +1,19 @@
 Template.submissions.helpers({
 
-  myHelper: function () {
-    //
+  userRating: function(applicationId) {
```
Comment (varffs) [inline app/client/templates/admin/submissions.js]: This is a duplicate. If generic put in the generic helpers file

Comment (varffs) [inline app/client/templates/admin/submissions.js]: Oh no it isn't quite. But could be refactored to a generic

**Pull request merged**